x-pack/plugins/ml/public/application/routing/breadcrumbs.ts

```diff
@@ -34,3 +34,10 @@ export const DATA_VISUALIZER_BREADCRUMB: ChromeBreadcrumb = Object.freeze({
   }),
   href: '#/datavisualizer',
 });
+
+export const CREATE_JOB_BREADCRUMB: ChromeBreadcrumb = Object.freeze({
+  text: i18n.translate('xpack.ml.createJobsBreadcrumbLabel', {
+    defaultMessage: 'Create job',
```

Comment:
Should this be "Create jobs" to reflect the fact that our modules/supplied configurations often create more than one job? I think that would only be possible if this breadcrumb is used only by the module path
```suggestion
    defaultMessage: 'Create jobs',
```